Quickstart/AzureSearchQuickstart/AzureSearchQuickstart.sln

```diff
@@ -0,0 +1,25 @@
+﻿
```

Comment:
It looks like in your attempt to rename directories, you ended up duplicating a bunch of files. Please fix it.